core/src/main/java/org/geysermc/geyser/translator/text/MessageTranslator.java

```diff
@@ -56,6 +57,9 @@ public class MessageTranslator {
     // Reset character
     private static final String RESET = BASE + "r";
 
+    // Pattern for matching formatting codes
+    private static final Pattern PATTERN = Pattern.compile("(?i)\\u00A7[0-9A-FK-OR]");
```

Comment:
Not using a regex here and using a `for` loop similar to the logic in convertMessage(Component, String) might be more efficient?

Reply:
Yep had just copied it from Minecraft, but should be 10 times fast now ^^